src/main/webapp/login/login.controller.js

```diff
@@ -22,6 +22,7 @@ angular.module('managementConsole')
         modelController.login(credentials.username, credentials.password).then(function () {
           $scope.authenticated = true;
           $scope.page.htmlClass = '';
+          // TODO: Why is this refreshing done here?
           var modelPromise = modelController.refresh();
```

Comment:
I had to redo how the whole model is loaded on login @Martinsos We can review that part a bit later but I suggest we leave refresh call here for now